components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -27,20 +27,22 @@ type Props = Pick<AIChatContext,
   | 'isToolsMenuOpen'
   | 'setIsToolsMenuOpen'
   | 'isMobile'
-  | 'hasInitialHistory'
   | 'shouldDisableUserInput'
   | 'hasAcceptedAgreement'>
 
 interface InputBoxProps {
     context: Props
     onFocusInputMobile?: () => unknown
+    maybeShowSoftKeyboard?: (querySubmitted: boolean) => unknown
```

Comment:
hey sorry, I know you're just cargo culting here but I think these methods should return `void` rather than unknown.

Even if we do return something from these functions its fine, it just means the `InputBox` doesn't do anything with the return type.

For example, the below is fine:

```ts
let foo: ((n: number) => void) = n => n * n
```

see playground:
https://www.typescriptlang.org/play/?ssl=5&ssc=28&pln=5&pc=29#code/C4TwDgpgBAUghiAzgMQK4DsDGUC8UAU6AXFOqgLYBGEATgJS4B8UAbgPYCWAJgFB8A2EYFABmbNiXhI0WXAXQMczdD0HDKcGpIQoM2PIUXKoAKlJA